package.json

```diff
@@ -0,0 +1,5 @@
+{
+	"devDependencies": {
+		"@pnpm/registry-mock": "^3.16.0"
```

Comment:
I don't see it being used anywhere. Should it be started by the user?

Reply:
I pushed it to wrong branch by mistake.